src/BookingForm.js

```diff
@@ -0,0 +1,153 @@
+import React, { useState } from "react";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCheckCircle } from "@fortawesome/free-solid-svg-icons";
+import "./BookingForm.css";
+
+const BookingForm = props => {
```

Comment:
Try deconstructing to make it easier to read
```suggestion
const BookingForm = ({ onAddBooking }) => {
```